src/compiler/types.ts

```diff
@@ -300,6 +300,12 @@ module ts {
         FirstNode = QualifiedName,
     }
 
+    export const enum StatementFlags {

```

Comment:
`StatementFlags` seems to only be used in `parser.ts`, so I don't think it warrants being in `types.ts`.